ctranslate2/calc_rtf.py

```diff
@@ -16,10 +16,10 @@
 ]
 
 n_batches = 3
-warmup_batches = 5
+warmup_batches = 3
 
-audio_file = "4469669.mp3"
-max_len = 600  # 10 minutes
+audio_file = "../data/sample_ami-es2015b.wav"
```

Comment:
Why change the audio file? What's wrong with the 4469669 file from EARNINGS 21?

Reply:
not ccby4, so we don;t have permissions to redistribute. AMI is CC-by-4

Reply:
Ah, good call there.